### PR TITLE
wandio 4.2.4-1

### PR DESCRIPTION
**Pull request by cho-m**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/wandio.rb
Comment (cho-m) [inline Formula/wandio.rb:4]: Original homepage (https://web.archive.org/web/20220305181150/https://research.wand.net.nz/software/libwandio.php) references wiki at https://github.com/wanduow/wandio/wiki. This redirects to new org.

Not sure about version number. `4.2.4-1` tag may be to match Debian equivalent of `revision`

**Review by iMichka — APPROVED** [review]

**Comment by iMichka** [comment]
Good for me. Why the do not merge?

Once this is merge we can try to build Ventura bottles.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**
